rdb/Datum/ObjectDatum.php

```diff
@@ -48,7 +48,11 @@ public function setValue($val)
 
     public function toNative($opts)
     {
-        $native = new \ArrayObject();
+        if (isset($opts['documentFormat']) && $opts['documentFormat'] == 'array') {
+            $native = array();
+        } else {
```

Comment:
I'd prefer an explicit set of allowed values. How about?
* `documentFormat` not set at all
* `documentFormat == 'array'`
* `documentFormat == 'object'`

So the else branch should verify that documentFormat is either not `isset`, or that its value is 'object'. Otherwise it should throw a `RqlDriverError`.

Does that sound reasonable to you?